buf.work.yaml

```
@@ -2,3 +2,5 @@ version: v1
 directories:
   - api/proto
   - proto
+  - lib/teleterm/api/proto
+  - lib/prehog/proto
```

nit: sort alphabetically:

```suggestion
  - api/proto
  - lib/prehog/proto
  - lib/teleterm/api/proto
  - proto
```